lib/bushslicer.rb

```diff
@@ -50,6 +50,7 @@ module BushSlicer
   else
     EXECUTOR_NAME = "#{HOSTNAME.split('.')[0]}-#{LOCAL_USER}".freeze
   end
+  EXECUTOR_NAME = "#{EXECUTOR_NAME}-e"
```

Comment:
What about `EXECUTOR_NAME = EXECUTOR_NAME.chomp('-')`, if the string ends with `-` then it's not modified at all.

Reply:
We can not guarantee `LOCAL_USER` end with a valid char for project name and workdir, too

Reply:
E.g, 
```
$ sudo useradd test-lxia-
[sudo] password for lxia: 

$ tail -3 /etc/passwd
systemd-timesync:x:975:975:systemd Time Synchronization:/:/sbin/nologin
flatpak:x:974:974:User for flatpak system helper:/:/sbin/nologin
test-lxia-:x:1001:1002::/home/test-lxia-:/bin/bash

$ ls /home/
lost+found  lxia  test-lxia-
```